PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,24 +2,196 @@
+```terminal
+$ python3 -m pip install -r requirements.txt
+```
+9. run the API locally on your machine
+```terminal
+$ gunicorn app.main:app -w 1 -k uvicorn.workers.UvicornWorker
+```
+Or
+```terminal
+uvicorn app.main:app --reload
+
+OR 
+'''terminal
+python -m app.main
+```
+8. close the app with control+c in terminal
+9. deactivate environment:
+```terminal
+$ deactivate
+```
+
+If you prefer to use pipenv and create a pipfile from our requirements.txt:
+1. clone the repo
+2. cd into repo
+3. add the saved model under your app folder
+4. create a .env file and ensure you have the following filled out correctly
+- CONSUMER_KEY = ???
+- CONSUMER_SECRET = ???
+- ACCESS_KEY = ???
+- ACCESS_SECRET = ???
+- DB_URL = ???
+
+MAP_API= ???
+BOT_NAME= Whateveryouwanttonameit
+
+5. install pip environment
+```terminal
+$ pipenv install
+```
+will create a pipfile for you
+6. activate the environment
+```terminal
+$ pipenv shell
+```
+7. run the API locally on your machine
+```terminal
+$ gunicorn app.main:app -w 1 -k uvicorn.workers.UvicornWorker
+```
+Or
+```terminal
+uvicorn app.main:app --reload
+```
+8. close the app with control+c in terminal
+9. deactivate environment:
+```terminal
+$ exit
+```
+
+# Windows:
+
+1. clone the repo
+2. cd into repo
+3. add the saved model under your app folder
+4. create a .env file and ensure you have the following filled out correctly
+- CONSUMER_KEY = ???
+- CONSUMER_SECRET = ???
+- ACCESS_KEY = ???
+- ACCESS_SECRET = ???
+- DB_URL = ???
+
+MAP_API= ???
+BOT_NAME= Whateveryouwanttonameit
+
+5. create virtual environment:
+```terminal
+$ py -m venv env
+```
+6. activate virtual environment:
+```terminal
+$ .\env\Scripts\activate
+```
+7. check activation:
+```terminal
+$ which python
+# should return:
+#   name_for_env/bin/python
+```
+
+8. install all dependencies with requirements.txt:
+```terminal
+$ py -m pip install -r requirements.txt
+```
+9. run the API locally on your machine
+```terminal
+uvicorn app.main:app --reload
+```
+10. close the app with control+c in terminal
+11. deactivate environment:
+```terminal
+$ deactivate
+```
+
+If you prefer to use pipenv and create a pipfile from our requirements.txt:
+1. clone the repo
+2. cd into repo
+3. add the saved model under your app folder
+4. create a .env file and ensure you have the following filled out correctly
+- CONSUMER_KEY = ???
+- CONSUMER_SECRET = ???
+- ACCESS_KEY = ???
+- ACCESS_SECRET = ???
+- DB_URL = ???
+
+MAP_API= ???
+BOT_NAME= Whateveryouwanttonameit
+
+5. install pip environment
+```terminal
+$ pipenv install
+```
+will create a pipfile for you
+6. activate the environment
+```terminal
+$ pipenv shell
+```
+7. run the API locally on your machine
+```terminal
+uvicorn app.main:app --reload
+```
+8. close the app with control+c in terminal
+9. deactivate environment:
+```terminal
+$ exit
+
+Once the application has been launced navigate to send form and fill out the fields to send a test post from the bot
```

Comment:
This is Great. We needed to update the documentation to reflect all the changes as well as the inefficiencies in the code. Including the fact that some of the code was indent with spaces while other code was indented with tabs.